src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/Configurations/Microbenchmarks.Configuration.cs

```diff
@@ -24,6 +24,7 @@ public sealed class Run : RunBase
     public class Environment
     {
         public uint default_max_seconds { get; set; } = 300;
+        public uint Iterations { get; set; } = 1;
```

Comment:
I don't see a corresponding implementation of the iteration mechanism in the ASP.NET command. Should we get rid of this until we have one?